src/redux/covidData/covidData.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+
+const FETCH_REQUEST = 'FETCH_REQUEST';
+const FETCH_SUCCESS = 'FETCH_SUCCESS';
+const FETCH_FAILURE = 'FETCH_FAILURE';
```

Comment:
- [Optional] It will be nice if we can scope actions based on the feature they represent. This will assist you in the future if you decide to scale your application.